ibc-testkit/src/testapp/ibc/clients/mock/client_state.rs

```diff
@@ -41,7 +41,7 @@ impl MockClientState {
     pub fn new(header: MockHeader) -> Self {
         Self {
             header,
-            trusting_period: Duration::from_nanos(0),
+            trusting_period: Duration::from_nanos(100),
```

Comment:
we can keep this untouched and use a separate method like `with_timestamp`.

Reply:
I see. In that case, can you please add a doc string for this?

Reply:
[cf24230](https://github.com/cosmos/ibc-rs/pull/1151/commits/cf242303c3d74b2c7ffdbba0b7f2ff148f29003a)